packages/api/test/repository-test.js

```diff
@@ -59,6 +59,47 @@ describe('Repository', () => {
     });
   });
 
+  describe('ticket handlers', async () => {
+    const state = 'testing-state';
```

Comment:
to ensure that multiple runs of the test suite can happen in parallel, I would make this a `uuid` just as you would in the normal workflow.